app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,2 @@
+class RecipesController < ApplicationController
+end
```

Comment:
I was told that we should use a namespace. Like `V1` This controller should be namespaced if that is correct.

Reply:
That is correct - we did agree on the namespace v1.

Reply:
Changed